etc/kayobe/inventory/group_vars/wazuh-manager/wazuh-manager

```diff
@@ -0,0 +1,153 @@
+---
+# Wazuh deployment details and documetation:
+# https://documentation.wazuh.com/current/deployment-options/deploying-with-ansible/index.html
+#
+# Defaults that are modified by this file:
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/ansible-filebeat-oss/defaults/main.yml
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/ansible-wazuh-manager/defaults/main.yml
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/wazuh-dashboard/defaults/main.yml
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/wazuh-indexer/defaults/main.yml
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/ansible-wazuh-manager/defaults/main.yml
+
+# Wazuh IP accessible from nodes running agents
+# Change network as needed
+wazuh_manager_ip: "{{ provision_oc_net_name | net_ip }}"
+
+# Single node setup
+single_node: true
+minimum_master_nodes: 1
+indexer_node_master: true
+
+# Configure hostnames for Wazuh indexer nodes
+# Example es1.example.com, es2.example.com
+domain_name: "dl"
```

Comment:
What is `dl`? Is there a sensible default we can provide here e.g. using a fact?